### ISSUE TITLE
Certificate must be exportable

### ISSUE THREAD

**Issue by AyrA** [webprofusion/certify#73]
For IIS 7 and older to work properly, the certificate must be marked as exportable. If not, any SSL connection will be aborted immediately and raise this error in the windows error log:

```
Protokollname: System
Quelle:        Schannel
Datum:         10.03.2017 17:54:38
Ereignis-ID:   36887
Aufgabenkategorie:Keine
Ebene:         Fehler
Schlüsselwörter:
Benutzer:      SYSTEM
Computer:      VM-MASTER
Beschreibung:
Es wurde eine schwerwiegende Warnung empfangen: 51.
Ereignis-XML:
```
```xml
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Schannel" Guid="{1F678132-5938-4686-9FDC-C8FF68F15C85}" />
    <EventID>36887</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>0</Task>
    <Opcode>0</Opcode>
    <Keywords>0x8000000000000000</Keywords>
    <TimeCreated SystemTime="2017-03-10T16:54:38.660238900Z" />
    <EventRecordID>194876</EventRecordID>
    <Correlation />
    <Execution ProcessID="512" ThreadID="4800" />
    <Channel>System</Channel>
    <Computer>VM-MASTER</Computer>
    <Security UserID="S-1-5-18" />
  </System>
  <EventData>
    <Data Name="AlertDesc">51</Data>
  </EventData>
</Event>
```

**Comment by webprofusion-chrisc** [comment]
Thanks for reporting this, this looks like a new problem in the 0.9.97 version released yesterday. Downloads have been temporarily suspended until I can investigate. We do request that the cert be exportable when adding it to the X509Store, if this not working then there's something else wrong somewhere.

**Comment by webprofusion-chrisc** [comment]
This has now been addressed with an update (V0.9.98). Re-using ACME Identifiers caused an issue with the private key mismatching the updated cert.

**Comment by webprofusion-chrisc** [comment]
Closing.
